README.md

```diff
@@ -1,4 +1,99 @@
 Sailjail permissions
 ====================
-
 Permissions for [Sailjail](https://github.com/sailfishos/sailjail/).
+These are [firejail](https://firejail.wordpress.com/) profile files with
+Sailjail specific metadata format.
+
+Permissions
+-----------
+Permissions that applications may use:
+
+- Accounts
+  - Using accounts, including editing them. Syncing accounts.
+- Ambience
+  - Set and edit ambiences.
+- AppLaunch
+  - Launching and stopping systemd services. This is usually needed for
+    background tasks.
+- ApplicationInstallation
+  - Installing and uninstalling applications.
+- Audio
+  - Playing audio, changing audio configuration and showing audio controls on
+    lockscreen.
+- Bluetooth
+  - Connecting to and using Bluetooth and NFC hardware.
+- Calendar
+  - Display and editing of calendar events.
+- Camera
+  - Access to camera hardware to take photos or video.
+- Contacts
+  - Display and editing of contacts data. Access to contact cards.
+- Documents
+  - Access to Documents directory.
+- Downloads
+  - Access to Downloads directory.
+- Email
+  - Reading and sending emails. Access to email attachments.
+- Internet
+  - Using data connection and connecting to internet.
+- Location
+  - Use GPS and low accuracy positioning.
+- MediaIndexing
+  - Access to Tracker to list files on device. If you have access to data
+    directory, you may want to use also this.
+- Messages
+  - Access to message data and to send SMS messages.
+- Mozilla
+  - If you use Gecko based WebView you need this.
```

Comment:
I thought this got renamed. But apparently it was just the translated description.

Kind of separate issue from this PR, but these filenames do become a part of interface -> they should be as correct as possible as changing becomes difficult after they are in active use.

And there is similar situation with few other files too: Connman, Location, SyncFW

Reply:
Yeap, similar